docs/guide/setup/creating_ligand_networks.rst

```diff
@@ -47,5 +46,8 @@ In the following code, we will show how a ``LigandNetwork`` can be planned:
    # Now let's plan the Network
    ligand_network = network_planner(ligands=mols, mappers=[mapper], scorer=scorer)
 
+More on generating ligand networks you can find in our generate_ligand_network_cookbook_ .
+.. _generate_ligand_network_cookbook: cookbook/generate_ligand_network
```

Comment:
```suggestion
Practical information on generating ligand networks can be found in our :ref:`cookbook for ligand network generation <cookbook_planning_ligandnetwork>` .
```
Slight text rewording - if we can merge https://github.com/OpenFreeEnergy/ExampleNotebooks/pull/124 first then we should be able to use this reference.